<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MCS</class>
 <widget class="QMainWindow" name="MCS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>780</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MCS(测控系统课程软件设计)</string>
  </property>
  <property name="accessibleName">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>采样波形</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QFrame" name="frame">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_15"/>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>采样波形FFT</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QFrame" name="frame_2">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_14"/>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>信号特征</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label">
                 <property name="minimumSize">
                  <size>
                   <width>74</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>最大幅值 (V)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit">
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="0" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <spacer name="horizontalSpacer_6">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>周期 (μs)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_3">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>137</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="minimumSize">
                  <size>
                   <width>74</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>频率 (kHz)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_2">
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="1">
              <spacer name="horizontalSpacer_7">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>输入电压选择</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="3" column="1">
              <widget class="QRadioButton" name="radioButton_2">
               <property name="text">
                <string>0 ~ 10 V</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QRadioButton" name="radioButton_4">
               <property name="text">
                <string>-10 ~ +10 V</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QRadioButton" name="radioButton_3">
               <property name="text">
                <string>-5 ~ +5 V</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QRadioButton" name="radioButton">
               <property name="text">
                <string>0 ~ 5 V</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>AD Timer</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>采样频率 (kHz)</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_4">
                 <property name="text">
                  <string>0.1</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_5">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QScrollBar" name="horizontalScrollBar">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>1000</number>
               </property>
               <property name="value">
                <number>1</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_7">
          <property name="title">
           <string>采样设置</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>开始采样/恢复数据</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="pushButton_2">
               <property name="text">
                <string>停止采样</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="pushButton_3">
               <property name="text">
                <string>静态显示</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="pushButton_9">
               <property name="text">
                <string>采样数据</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="title">
           <string>波形设置</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="2">
              <widget class="QPushButton" name="pushButton_5">
               <property name="text">
                <string>缩放</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="pushButton_6">
               <property name="text">
                <string>拖动</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QPushButton" name="pushButton_4">
               <property name="text">
                <string>还原采样波形</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="pushButton_8">
               <property name="text">
                <string>还原FFT</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="pushButton_7">
               <property name="text">
                <string>保存</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QPushButton" name="pushButton_10">
               <property name="text">
                <string>频率设置</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>797</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_Recently"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionExport_Data"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>View(&amp;V)</string>
    </property>
    <widget class="QMenu" name="menuColor_Setting">
     <property name="title">
      <string>Color Setting</string>
     </property>
     <addaction name="actionBackgroud_Color"/>
     <addaction name="actionLine_Color"/>
     <addaction name="actionPoint_Color"/>
    </widget>
    <widget class="QMenu" name="menuSignal">
     <property name="title">
      <string>Signal</string>
     </property>
     <addaction name="actionSquare"/>
     <addaction name="actionTriangle"/>
     <addaction name="actionSin"/>
    </widget>
    <addaction name="menuSignal"/>
    <addaction name="menuColor_Setting"/>
    <addaction name="actionStyle_Sheet"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
    <addaction name="actionsome_help"/>
    <addaction name="separator"/>
    <addaction name="actionabout_some"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="actionsome_help">
   <property name="text">
    <string>MCS Help</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+/</string>
   </property>
  </action>
  <action name="actionabout_some">
   <property name="text">
    <string>About MCS</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionOpen_Recently">
   <property name="text">
    <string>Open Recent</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionBackgroud_Color">
   <property name="text">
    <string>Backgroud Color</string>
   </property>
  </action>
  <action name="actionLine_Color">
   <property name="text">
    <string>Line Color</string>
   </property>
  </action>
  <action name="actionPoint_Color">
   <property name="text">
    <string>Point Color</string>
   </property>
  </action>
  <action name="actionExport_Data">
   <property name="text">
    <string>Export Data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSquare">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Square</string>
   </property>
  </action>
  <action name="actionTriangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Triangle </string>
   </property>
  </action>
  <action name="actionSin">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Sine</string>
   </property>
  </action>
  <action name="actionStyle_Sheet">
   <property name="text">
    <string>Style Sheet</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
